cfme/tests/configure/test_default_filters.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+
+
+import pytest
+from cfme.configure import settings as st
+from cfme.fixtures import pytest_selenium as sel
+from utils.providers import setup_a_provider
+from cfme.cloud import instance  # NOQA
+
+
+@pytest.fixture(scope="module")
+def setup_first_provider():
+    setup_a_provider(validate=True, check_existing=True)
+
+
+@pytest.mark.parametrize('filters',
+                        [[['Cloud', 'Instances', 'Images', 'Platform / Amazon']]])
+def test_cloudimage_defaultfilters(filters, setup_first_provider):
+    df = st.DefaultFilter(name='Platform / Amazon')
+    df.update({'filters': [(k, True) for k in filters]})
+    sel.force_navigate('clouds_images_filter_folder', context={'folder_name': 'Global Filters'})
+    assert sel.is_displayed_text(df.name), "Default Filter settings Failed!"
+
+
+@pytest.mark.parametrize('filters',
+                        [[['Cloud', 'Instances', 'Instances', 'Platform / Openstack']]])
```

Comment:
I think it does need to be that way for the tests to work like this, since (I think?) removing one of the containing lists would make py.test parametrize on each filter. 

That said, I think this method is a midpoint between two extremes. Defining the filters inside the tests and not parametrizing at all is one extreme, and the other extreme would be to parametrize everything, including the `DefaultFilter` initializer arg, nav targets and context, etc.

I think I'd prefer seeing the parametrization go away, in favor of defining `filters` in the test body.